artemis/modules/nuclei_dast.py

```diff
@@ -0,0 +1,435 @@
+#!/usr/bin/env python3
+import collections
```

Comment:
I don't think copying the whole file is a good idea - it will make it very easy to make mistakes, where a fix is applied in one but not the second one. What about inheritance or any other way of extracting common code into one place?